projects/rawkode-academy/web/package.json

```diff
@@ -60,6 +60,9 @@
     "endpointId": "rawkode-academy-Gpkz"
   },
   "trustedDependencies": [
-    "@biomejs/biome"
+    "@biomejs/biome",
+    "esbuild",
```

Comment:
Why do we need esbuild, sharp, and workerd?

Reply:
This got automatically added when I was running `bun install` and since we have the `trustedDependencies` specified it won't run the lifecycle hooks of untrusted dependencies anymore. At least that's what I saw. I added all deps via `bun pm trust --all` and this is the list of `trustedDependencies` now